mypyc/codegen/emitfunc.py

```diff
@@ -426,6 +427,24 @@ def visit_get_attr(self, op: GetAttr) -> None:
             elif not always_defined:
                 self.emitter.emit_line("}")
 
+    def visit_get_attr_nullable(self, op: GetAttrNullable) -> None:
```

Comment:
If you implement the suggestion to not use a subclass, it still makes sense to keep this code in a separate method that is called from the `visit_get_attr` handler if the flag is set.